### PR TITLE
Pets topic bundle and various tweaks

### PR DESCRIPTION
**Pull request by Blizzardsev**
This will introduce the following:

- Two new pet-related topics; `talk_did_you_have_pets `and `talk_service_animals`.
- Tweaks to `screenshots.rpy` and the corresponding topics `talk_having_pictures_taken`, `talk_get_picture_permission` to take advantage of new affinity code and general housekeeping.
- Replacement of vanilla menu highlight/confirm sounds to a stopgap based off Natsuki's Okay Everyone! (Until original sounds can be composed).

### CODE REVIEW

File: game/script-topics.rpy [hunk 1]
```diff
@@ -294,22 +312,283 @@ label talk_get_picture_permission:
             $ natsuki_approves = random.randint(1, 100) <= ((100 - bad_screenshot_streak) * 25)
```
Comment (spaghetsie) [inline game/script-topics.rpy:312]:
```suggestion
            $ natsuki_approves = random.randint(1, 100) <= ((100 - jn_screenshots.bad_screenshot_streak) * 25)
```

bad_screenshot_streak wasn't accessible by itself

File: game/script-topics.rpy [hunk 2]
```diff
@@ -294,22 +312,283 @@ label talk_get_picture_permission:
...
+            $ persistent.jn_player_pet = "fish"
+
+        "Gerbils":
+            n "Awww!{w=0.2} I like gerbils!"
+            n "It's so cute how they live in little groups to keep each other company."
+            n "They're good at digging,{w=0.1} too -{w=0.2} like seriously good!"
+            n "Take good care of yours for me,{w=0.1} okay?"
+            $ persistent.jn_player_pet = "gerbils"
+
+        "More...":
```
Comment (spaghetsie) [inline game/script-topics.rpy:448]: (just for future reference - this should be removed once new menus are in place)

**Review by spaghetsie — CHANGES_REQUESTED** [review]
(first review ever so bear with me if I messed up something)

Looks great!
One tiny little thing was causing crashes but no big deal

File: game/script-topics.rpy (hunk 1 above)
Comment (spaghetsie) [reply game/script-topics.rpy:312]:
```suggestion
            $ natsuki_approves = random.randint(1, 100) <= (100 - (bad_screenshot_streak * 25))
```
Actually I also wonder if this wasn't the intended formula

Comment (Blizzardsev) [reply game/script-topics.rpy:312]: My bad! I remember fiddling with this formula a bit, and must have misplaced some brackets when tidying up after verifying it worked. Thank you!

File: game/script-topics.rpy (hunk 2 above)
Comment (Blizzardsev) [reply game/script-topics.rpy:448]: Will do!

**Review by spaghetsie — APPROVED** [review]
_patpat_

**Pull request merged**